lib/domUpdates.js

```diff
@@ -1,19 +1,37 @@
 const domUpdates = {
 
-  renderScoreboard(players) {
-    players.forEach((player) => {
+  renderScoreboard (players) {
+    players.forEach(player => {
       let ul = document.querySelector('.scoreboard');
-      let li = document.createElement('li');
-      ul.appendChild(li)
-      li.innerHTML = `Player: ${player.name}
-                      <br>
-                      Score: ${player.score}`;
+      let div = document.createElement('div');
+      ul.appendChild(div)
+      div.innerHTML = `<li>Player: ${player.name}</li>
+                      <li>Score: ${player.score}</li>`;
+    })
+  },
 
+  convertCategoryNames(camelCaseNames) {
+    return camelCaseNames.map(name => {
+      let result = name.replace( /([A-Z])/g, " $1" );
+      let finalResult = result.charAt(0).toUpperCase() + result.slice(1);
+      return finalResult;
```

Comment:
Little difficult to follow what's happening here. `result` and `finalResult` aren't the most descriptive variable names. Is there something you could rename these to in order to make the logic a bit clearer?